src/app/modules/main/communities/minting/view.nim

```diff
@@ -18,8 +18,8 @@ QtObject:
     result.QObject.setup
     result.mintingModule = mintingModule
 
-  proc mintCollectible*(self: View, communityId: string, fromAddress: string, name: string, symbol: string, description: string, supply: int, infiniteSupply: bool, transferable: bool, selfDestruct: bool, network: string) {.slot.} =
-    self.mintingModule.mintCollectible(communityId, fromAddress, name, symbol, description, supply, infiniteSupply, transferable, selfDestruct, network)
+  proc mintCollectible*(self: View, communityId: string, fromAddress: string, name: string, symbol: string, description: string, supply: int, infiniteSupply: bool, transferable: bool, selfDestruct: bool, chainId: int) {.slot.} =
```

Comment:
Hm.. so this looks like somehow in the view one has to specify the chain ID in the minting process.
I think we should rather rely on whatever chain we're connected to at the moment.

So instead of passing the chain ID here, retrieve the currently connected-to chainId from some service when performing the actual mint.

Another thing I just noticed now:
Isn't `mintCollectible` semantically incorrect? What this method does is deploying a contract via status-go no?
Minting will happen later when someone actually performs a mint on the already deployed contract.

Anyways just spitballing here. This might lead to confusion

Reply:
The owner chooses which chain he deploys to. There is a combo box to choose chain during deployment process.
Re: mint word, yes I was thinking about this. In UI the name is "mint", however it is deployment operation.
I will change it in one of next prs.

Reply:
If that is the desired way to do this, then it's okay